src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -351,39 +351,37 @@ const ParcelsPage: React.FC<{}> = () => {
                             router.push("/parcels");
                         }}
                         headerId="expandedParcelDetailsModal"
+                        footer={
+                            <Centerer>
+                                <LinkButton link={`/parcels/edit/${selectedParcelId}`}>
+                                    Edit Parcel
+                                </LinkButton>
+                                {selectedClientDetails && (
+                                    <>
+                                        <LinkButton
+                                            link={`/clients?clientId=${selectedClientDetails.clientId}`}
+                                        >
+                                            See Client Details
+                                        </LinkButton>
+                                        <LinkButton
+                                            link={`/clients/edit/${selectedClientDetails.clientId}`}
+                                        >
+                                            Edit Client Details
+                                        </LinkButton>
+                                    </>
+                                )}
+                            </Centerer>
+                        }
                     >
                         <OutsideDiv>
                             <ContentDiv>
                                 <Suspense fallback={<ExpandedParcelDetailsFallback />}>
                                     <ExpandedParcelDetails parcelId={selectedParcelId} />
                                 </Suspense>
                             </ContentDiv>
-                            <ButtonsDiv>
-                                <Centerer>
-                                    <LinkButton link={`/parcels/edit/${selectedParcelId}`}>
-                                        Edit Parcel
-                                    </LinkButton>
-                                    {selectedClientDetails && (
-                                        <>
-                                            <LinkButton
-                                                link={`/clients?clientId=${selectedClientDetails.clientId}`}
-                                                disabled={!selectedClientDetails.isClientActive}
```

Comment:
I think we should keep these?

Reply:
Yes sorry I've added them back in the buttons